services/web/server/src/simcore_service_webserver/activity/module_setup.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from aiohttp import web
+from servicelib.application_setup import ModuleCategory, app_module_setup
+from servicelib.rest_routing import iter_path_operations, map_handlers_with_operations
+
+from ..rest_config import APP_OPENAPI_SPECS_KEY
+from . import handlers
+from .config import assert_valid_config
+
+logger = logging.getLogger(__name__)
+
+
+@app_module_setup(
+    "simcore_service_webserver.activity",
+    category=ModuleCategory.ADDON,
+    depends=["simcore_service_webserver.rest"],
+    logger=logger,
+)
+def setup_activity(app: web.Application):
+
+    #----------------------------------------------
+    # TODO: temporary, just to check compatibility between
+    # trafaret and pydantic schemas
+    assert_valid_config(app)
+    #---------------------------------------------
```

Comment:
Could a nice warning be added here as well to remind our future selves that this must be removed?

Reply:
Good point but I will  remove this right away in iteration 2. I just needed to get this first iteration in master so that I can also reuse it somewhere else